### PR TITLE
ASTRACTL-32896: fix namespace yaml situation

### PR DESCRIPTION
**Pull request by palicena**
This PR does a few things:
- Adds back the namespace resource to `neptune.yaml` to unblock the polaris qual pipeline (would have to wait for a neptune PR to merge otherwise)
- Separates the namespace resource in `details/operator-sdk/config/default/manager.yaml` into its own folder so that it can be excluded from `cluster-install/kustomization.yaml`
- Add a new `details/operator-sdk/config/unified-install-script/kustomization.yaml` that gets pulled in by the cluster-install kustomization file
- Update relevant kustomization files

### CODE REVIEW

File: details/operator-sdk/config/unified-installer-script/kustomization.yaml
Comment (palicena) [inline details/operator-sdk/config/unified-installer-script/kustomization.yaml:7]: @nhammern I did end up creating a new folder with a kustomization file otherwise I would've had to either add `namePrefix: operator-` to the cluster-install kustomization (which would've screwed up the neptune.yaml names) or modified the install script to take into account the lack of the prefix.

File: cluster-install/neptune.yaml
Comment (palicena) [inline cluster-install/neptune.yaml:13]: Re-added this to unblock polaris qual pipeline (would've had to wait for a neptune PR merge otherwise).

**Review by oscarr-ntap — APPROVED** [review]

**Review by nhammern — APPROVED** [review]

**Pull request merged**
